src/attendance/dto/attendance-search.dto.ts

```diff
@@ -7,6 +7,12 @@ export class AttendanceSearchDto {
   })
   limit: string;
 
+  @ApiProperty({
```

Comment:
Do you see a use case to search the attendance database by thr record id ? Most use cases would be needing to fetch records for a specific date range, for a given class etc

Reply:
Fixed 👍